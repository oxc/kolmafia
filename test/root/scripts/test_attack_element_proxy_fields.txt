ash $monster[warwelf].attack_element
ash $monster[warwelf].attack_elements
ash $monster[Knott Yeti].attack_element
ash $monster[Knott Yeti].attack_elements
ash $monster[lihc].attack_element
ash $monster[lihc].attack_elements
js Monster.get('warwelf').attackElement
js Monster.get('warwelf').attackElements
js Monster.get('Knott Yeti').attackElement
js Monster.get('Knott Yeti').attackElements
js Monster.get('lihc').attackElement
js Monster.get('lihc').attackElements