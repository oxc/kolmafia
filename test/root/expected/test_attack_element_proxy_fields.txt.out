Returned: none
image => circle.gif
Returned: aggregate boolean [element]
none => true
Returned: cold
image => snowflake.gif
Returned: aggregate boolean [element]
cold => true
Returned: spooky
image => skull.gif
Returned: aggregate boolean [element]
sleaze => true
spooky => true
Returned: none
image => circle.gif
Returned: aggregate element [1]
0 => none
Returned: cold
image => snowflake.gif
Returned: aggregate element [1]
0 => cold
Returned: spooky
image => skull.gif
Returned: aggregate element [2]
0 => sleaze
1 => spooky